electron-app/main.js

```diff
@@ -21,6 +21,10 @@ app.commandLine.appendSwitch('disable-renderer-backgrounding');
 app.commandLine.appendSwitch('disable-background-timer-throttling');
 
 process.env.PORT = process.env.PORT || 9247;
+// Allow override if using a staging/dev server.
+if (!process.env.ARTCONOMY_URL) {
```

Comment:
Not terribly keen on setting environment dependencies here.
At most it should be pulled into its own file for handling situations like this.
And hardcoding the url through this method is undesirable.

At best the implementing service should use base_urk = <env setting> || '<default url>'